generators/app/prompts.js

```diff
@@ -9,6 +13,21 @@ module.exports = {
           name: 'seedbank',
           checked: true,
           value: "gem 'seedbank'"
+        },
+        {
+          name: 'rack-cors',
+          checked: true,
+          value: "gem 'rack-cors'"
+        },
+        {
+          name: 'redis',
+          checked: true,
+          value: "gem 'redis', '~> 3.0'"
+        },
+        {
+          name: 'bcrypt',
+          checked: true,
+          value: "gem 'bcrypt', '~> 3.1.7'"
```

Comment:
More on more: I'm stating to prefer a simple solution over magic, this is why `minitest` wash choosed.

In combination with pure functions a simple assert is what you need 90% of the time. If don't, something smells wrong. Ruby is becoming a problem in that manner, it's a limited language of functional constructs, more one reason to decouple things.